src/components/SplitDialog.vue

```diff
@@ -617,47 +682,98 @@ export default {
   }
 }
 
-.toolbar-title {
-  width: 107px;
-  height: 20px;
-  color: $app-primary-color;
-  font-size: 14px;
-  font-weight: normal;
-  line-height: 20px;
-  margin-left: 11px;
-  margin-top: 4px;
-}
+.toolbar-flex-container {
+  display:flex;
+  flex-direction: row;
+
+  .select-box {
+    width: 200px;
+    border-radius: 4px;
+    border: 1px solid rgb(144, 147, 153);
+    background-color: $background;
+    font-weight: 500;
+    color: rgb(48, 49, 51);
+    margin-left: 8px;
+    margin-top: 3px;
+    margin-bottom: 2px;
+
+    ::v-deep .el-input__inner {
+      width:177px;
+      color: $app-primary-color;
+      height: 24px;
+      padding-left: 4px;
+      padding-right: 8px;
+      background-color: $background;
+      border-style: none;
+    }
+    
+    ::v-deep .el-input__icon {
+      line-height: 24px;
+      color: $lightGrey;
+    }
+  }
 
-.select-box {
-  width: 180px;
-  border-radius: 4px;
-  border: 1px solid rgb(144, 147, 153);
-  background-color: var(--white);
-  font-weight: 500;
-  color: rgb(48, 49, 51);
-  margin-left: 8px;
-  margin-top: 2px;
-
-  ::v-deep .el-input__inner {
-    color: $app-primary-color;
+  i .select-box ::v-deep .el-input__icon {
+    color: rgb(48, 49, 51);
     height: 24px;
     padding-left: 8px;
     padding-right: 8px;
   }
-  ::v-deep .el-input__icon {
-    line-height: 24px;
+
+  .text {
+    margin-left: 8px;
+    margin-top: 7px;
+    font-weight: 500;
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
   }
 
-  ::v-deep .is-focus .el-input__inner {
-    border: 1px solid $app-primary-color;
+  .title {
+    width: 140px;
+    color: $app-primary-color;
   }
-}
 
-i .select-box ::v-deep .el-input__icon {
-  color: rgb(48, 49, 51);
-  height: 24px;
-  padding-left: 8px;
-  padding-right: 8px;
+  .search-text {
+    margin-top: 8px;
+    color: $grey;
+    font-size: 14px;
+    margin-left: 1rem;
+  }
+
+  .search-box {
+    margin-top: 2px;
+    margin-left:0.5rem;
+    height:28px;
+    width:137px;
+    padding-top:1px;
+    ::v-deep .el-input__inner {
+      background-color: $background;
+      height:28px;
+      line-height:28px;
+      border: 1px solid rgb(144, 147, 153);
+      border-radius: 4px;
+      &:focus {
+        border-color: $app-primary-color;
+      }
+    }
+  }
+
+  .magnify {
+    margin-top: 1px;
+    margin-left:0.5rem;
+    background: #8300bf;
+    border: 1px solid #8300bf;
```

Comment:
To get the search icon the same size as the search text, either:

 - remove the border
 - reduce the width and height of icon by 2px
 
 Before:
 
![image](https://user-images.githubusercontent.com/37255664/163300585-9552e273-cf52-46b0-b195-80b5b1ff6a17.png)

  
 After:
 
![image](https://user-images.githubusercontent.com/37255664/163300550-cf1a2586-5a18-477b-b5a0-e77a43abaf08.png)